src/index.tsx

```diff
@@ -91,6 +91,13 @@ console.log("Welcome to Simularium " + SIMULARIUM_BUILD_ENVIRONMENT + " build");
 console.log("Simularium Website Version " + SIMULARIUM_WEBSITE_VERSION);
 console.log("Simularium Viewer Version " + SIMULARIUM_VIEWER_VERSION);
 
-const renderApp = () => render(<App />, document.getElementById(APP_ID));
+const renderApp = (container: HTMLElement) => {
+    const root = createRoot(container);
+    root.render(<App />);
+};
+
+const container: HTMLElement | null = document.getElementById(APP_ID);
 
-renderApp();
+container !== null
+    ? renderApp(container)
+    : console.log("Could not find container element with id " + APP_ID);
```

Comment:
re: your comment about the check here, this to me suggests that there is a possibility of the container being null, when it should (theoretically) never be. I'd slightly prefer the `!` syntax because it seems like hitting an error will be more informative than the blank screen and a console message.

Reply:
trying this instead:
```
const container: HTMLElement | null = document.getElementById(APP_ID);
// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
const root = createRoot(container!);
root.render(<App />);
```